boxes/groups/services/storage-dapp-service/test/storage.spec.js

```diff
@@ -141,14 +169,20 @@ describe(`LiquidStorage Test`, () => {
         const permission = "active";
         const path = "test/utils/YourTarBall.tar";
         const content = fs.readFileSync(path);
-        const result = await storageClient.upload_public_file(
```

Comment:
This just did a normal file upload of the `.tar` file before instead of an actual archive + extract upload